internal/template/character/stam.go

```diff
@@ -122,6 +164,31 @@ func (c *Character) Jump(p map[string]int) (action.Info, error) {
 	}, nil
 }
 
+func (c *Character) JumpLength() int {
+	if c.Core.Player.LastAction.Type == action.ActionDash {
+		switch c.CharBody {
+		case info.BodyLoli, info.BodyLady, info.BodyMale:
+			return 37
+		case info.BodyGirl, info.BodyBoy:
+			return 34
+		default:
+			return 36
```

Comment:
Please use one of the other two cases as default since the 36 is actually not a frame count for any of the models above.